test-browser/simpleContract.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+var fs = require('fs');
+
+module.exports = {
+  'Simple Contract': function (browser) {
+    initTestData(function (error, testData) {
+      if (error) {
+        console.log(error);
+      } else {
+        runTests(browser, testData);
+      }
+    });
+  }
+};
+
+function runTests (browser, testData) {
+  browser
+    .url('http://127.0.0.1:8080/#version=builtin')
+    .waitForElementVisible('.newFile', 10000)
+  browser.assert.notEqual(testData, null)
+  /*
+  TODO push contracts to the editor and test
+  setEditorContent(browser, testData.testSimpleContract.sources.Untitled, function () {
+    getCompiledContracts(browser, function (contracts) {
```

Comment:
I think you need some kind of wait loop here. I'm not sure if my solution in `compiling.js` works, but you could take a look.